modules/apps/static/k8s-agent/k8s-agent-impl/build.gradle

```diff
@@ -0,0 +1,47 @@
+dependencies {
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.13.2"
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-core", version: "2.13.2"
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-databind", version: "2.13.2.2"
+	compileInclude group: "com.fasterxml.jackson.dataformat", name: "jackson-dataformat-yaml", version: "2.13.2"
+	compileInclude group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310", version: "2.13.2"
+	compileInclude group: "com.github.mifmif", name: "generex", version: "1.0.2"
+	compileInclude group: "com.squareup.okhttp3", name: "logging-interceptor", version: "3.12.12"
+	compileInclude group: "com.squareup.okhttp3", name: "okhttp", version: "3.12.12"
+	compileInclude group: "com.squareup.okio", name: "okio", version: "1.15.0"
+	compileInclude group: "dk.brics.automaton", name: "automaton", version: "1.11-8"
+	compileInclude group: "io.fabric8", name: "kubernetes-client", version: "5.12.2"
```

Comment:
@rotty3000 not sure what's our common practice with build.gradle files, but do we extract the version for libraries we want to keep in sync across multiple dependencies? For instance, would we have 5.12.2 as the value for a constant in the file for ${KUBERNETES_CLIENT_VERSION}?